notifications/06-13-20-0900.md

```diff
@@ -0,0 +1,26 @@
+**What is happening?**
+
+RFC 194163 
+
+CITZ - Standard Change #33 MCS PROD - MCS OS Patching to PROD cluster and NFS server
+
+**When?**
+
+Starting at Saturday June 13th @ 0800 and finishing at Friday June 19th @ 1600.
+
+**Will there be an impact on the Platform apps?**
+
+Some infrastructure services like metrics and Kibana will be briefly unavailable or have reduced performance during some operations. The app nodes will be drained in sequence during the weekdays and pods may experience some brief outages as pods are moved between nodes.
+
+**Do I need to do anything?**
+
+Monitor the health of your apps during the change. Refer to [Post-Outage Application Checklist](https://developer.gov.bc.ca/Post-Outage-Application-Health-Checklist) for health check tips.
```

Comment:
Ensure your app is designed in accordance with [Application Resiliency Guidelines](https://developer.gov.bc.ca/Resiliency-Guidelines) and is capable of withstanding brief outages.